vertx-web-graphql/pom.xml

```diff
@@ -78,6 +106,28 @@
         </plugin>
       </plugins>
     </pluginManagement>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-failsafe-plugin</artifactId>
+        <version>3.0.0-M7</version>
+        <configuration>
+          <includes>
+            <include>ValidationTest.java</include>
+            <include>BasicIT.java</include>
+            <include>ExampleIT.java</include>
```

Comment:
I believe it's possible to include any test which name ends with `IT.java`. Can you verify? You should rename `ValidationTest` then.